resources/assets/uad_lists.json

```
@@ -3548,7 +3548,7 @@
   {
     "id": "com.samsung.android.app.watchmanagerstub",
     "list": "Oem",
-    "description": "Stub for the watch manager. See above\nStub = https://stackoverflow.com/questions/10648280/what-is-stub-and-aidl-for-in-java\n",
+    "description": "Stub for the Galaxy Wearable app.\nStub = https://stackoverflow.com/questions/10648280/what-is-stub-and-aidl-for-in-java\n",
```

```suggestion
    "description": "Stub for the Galaxy Wearable app.\nStub = https://stackoverflow.com/questions/10648280/what-is-stub-and-aidl-for-in-java",
```

You don't need to end with an extra line.

---

I didn't touch the line endings myself 😄
They were here before I made some changes. 

Nevertheless,  I will add the changes 👍